### DISCUSSION TITLE
Panel: Override Time Range

### DISCUSSION THREAD

**Discussion by baur** [grafana/grafana#54560] [Dashboards, Panels & Visualizations]
How to Override specific Time Ranges for panel

I can do it to: tody, this month, etc

![image](https://user-images.githubusercontent.com/2898361/187826284-03efbceb-5239-48ae-9e69-ae27c81b2815.png)

But I'd like to set the next specific Ranges (technological shifts):

00:00 - 08:00
08:01 - 16:00
16:01 - 23:59

any suggestions?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
